lib/utils/h1_helper.ts

```diff
@@ -53,6 +53,16 @@ export class H1Helper {
                 processTimeOfDeparture(decodeResult, fields[i].substring(2).split(',')); // Strip off TD
             } else if (fields[i].startsWith('FX')) {
                 ResultFormatter.freetext(decodeResult, fields[i].substring(2));
+            }  else if (fields[i].startsWith('ET')) {
+                if(fields[i].length === 7) { // 1 digit day
+                //ResultFormatter.dayOfMonth(decodeResult, Number(fields[i].substring(1, 3)));
+                ResultFormatter.eta(decodeResult, DateTimeUtils.convertHHMMSSToTod(fields[i].substring(3)+'00'));
```

Comment:
will make changes - my guestion is what do we want to do with the day because we don't have the month or the year? Store it as `raw.eta_day` and add it as a result field? I don't think it belongs in `raw.day_of_month` because it's an estimate. currently i'm dropping it, but that seems silly.